modules/programs/gpg.nix

```diff
@@ -30,6 +32,15 @@ in {
         <link xlink:href="https://gnupg.org/documentation/manpage.html"/>.
       '';
     };
+
+    keyfiles = mkOption {
+      type = types.listOf types.path;
+      example = [ ./pubkeys.txt ];
+      default = [ ];
+      description = ''
+        A list of keyfiles to be imported into GnuPG.
```

Comment:
This should include a clear warning that the listed files will be copied in a world-readable fashion to the Nix store.